### DISCUSSION TITLE
create-nillion-app is git-unaware

### DISCUSSION THREAD

**Discussion by a3s7p** [NillionNetwork/discussions#129] [Bugs]
When setting up a project with [create-nillion-app](https://github.com/NillionNetwork/create-nillion-app), there is no .gitignore and some directories are created empty which makes Git skip them when committing to a repo.

Suggestions:
- add a .gitignore, perhaps based on the [official GitHub one](https://github.com/github/gitignore/blob/main/Node.gitignore) for Node
- also add Nada Python venv to .gitignore and make sure it's initialized if missing
- inside the empty directories, touch a `.keep` or `.gitkeep` empty file

**Comment by crypblizz8** [comment]
Thanks for the catch @a3s7p  - will be updated in our next PR. Thanks!
